### PR TITLE
remove explicit colors in favor of semantic formatters

### PR DESCRIPTION
**Pull request by mdrohmann**
https://www.pivotaltracker.com/story/show/174262019

### CODE REVIEW

File: internal/runners/run/run.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func run(out output.Outputer, subs subshell.SubShell, proj *project.Project, nam
 	if !lang.Recognized() {
 		warning := locale.Tl(
 			"run_warn_deprecated_script_without_language",
-			"[YELLOW]DEPRECATION WARNING: Scripts without a defined language currently fall back to using  the default shell for your platform. This fallback mechanic will soon stop working and a language will need to be explicitly defined for each script. Please configure the 'language' field with a valid option (one of {{.V0}})[/RESET]",
+			"[NOTICE][HIGHLIGHT]DEPRECATION WARNING:[/RESET][NOTICE] Scripts without a defined language currently fall back to using  the default shell for your platform. This fallback mechanic will soon stop working and a language will need to be explicitly defined for each script. Please configure the 'language' field with a valid option (one of {{.V0}})[/RESET]",
```
Comment (Naatan) [inline internal/runners/run/run.go:86]:
```suggestion
			"[NOTICE][HIGHLIGHT]DEPRECATION WARNING:[/RESET][NOTICE] Scripts without a defined language currently fall back to using  the default shell for your platform. This fallback mechanic will soon stop working and a language will need to be explicitly defined for each script. Please configure the 'language' field with a valid option (one of {{.V0}})[/RESET]",
```

Is this covered in the UX mocks? ie. using HIGHLIGHT here at all.

File: internal/output/color.go [hunk 2]
```diff
@@ -12,12 +12,62 @@ var colorRx *regexp.Regexp
...
+
+// Heading switches to bold and bright foreground
+func (dct defaultColorTheme) Heading(writer io.Writer) {
+	ct.Foreground(writer, ct.White, true)
+	ct.ChangeStyle(writer, ct.Bold)
+}
+
+// Notice switches to bright foreground
+func (dct defaultColorTheme) Notice(writer io.Writer) {
+	ct.Foreground(writer, ct.White, true)
```
Comment (Naatan) [inline internal/output/color.go:41]: Isn't this also setting it as white? The AC just calls for "bright".

**Review by Naatan — CHANGES_REQUESTED** [review]

File: internal/output/color.go (hunk 2 above)
Comment (mdrohmann) [reply internal/output/color.go:41]: Unfortunately, it is not possible to just set the "bright" flag, you always have to specify a color, and white is the default color.  When I set "white+bright" on my terminal which has a solarized color theme, it actually shows up as bright black (or grey-ish).  So, I think that's the best we can do here.

File: internal/runners/run/run.go (hunk 1 above)
Comment (mdrohmann) [reply internal/runners/run/run.go:86]: You are probably right. It just says

> great for highlights and actionable content

But it is not really actionable.  So, I think I will remove the HIGHLIGHT.

File: internal/output/color.go (hunk 2 above)
Comment (Naatan) [reply internal/output/color.go:41]: I see. In that case let's merge `HIGHLIGHT` into `NOTICE`. So let's drop `HIGHLIGHT` and instead make all those instances use `NOTICE`.

**Review by Naatan — CHANGES_REQUESTED** [review]
See my response to your comment

File: internal/output/color.go (hunk 2 above)
Comment (mdrohmann) [reply internal/output/color.go:41]: Just to make sure that I undestand correctly:  Would you like to remove the HIGHLIGHT formatter completely (although it is using a different color `teal`)?

I exchanged some uses of `[BOLD] a url or something [/RESET]` with highlight which seems sensible to me.  Do you want me to change these to `[NOTICE]` or remove them completely?

Comment (mdrohmann) [reply internal/output/color.go:41]: Mostly `[HIGHLIGHT]` is currently used for State Tool commands that could be copied and pasted to the command line.

Comment (Naatan) [reply internal/output/color.go:41]: Ok, let's maybe just make the difference more distinct then. Let's rename `HIGHLIGHT` to `ACTIONABLE`.

File: internal/output/color.go [hunk 3]
```diff
@@ -107,7 +107,7 @@ func colorize(ct ColorTheme, writer io.Writer, arg string) {
 		ct.Error(writer)
 	case `DISABLED`:
 		ct.Disabled(writer)
-	case `HIGHLIGHT`:
+	case `ACTIONABLE`:
 		ct.Highlight(writer)
```
Comment (Naatan) [inline internal/output/color.go:111]: Can we rename this too please

**Review by Naatan — CHANGES_REQUESTED** [review]

File: internal/output/color.go (hunk 3 above)
Comment (mdrohmann) [reply internal/output/color.go:111]: Ah! Thanks for catching.

**Review by Naatan — APPROVED** [review]

**Pull request merged**